public/scripts/presetScript.js

```diff
@@ -0,0 +1,99 @@
+const presetBox = document.querySelector("#preset-box");
+
+async function savePreset() {
+    // TODO: Add game theme
+    let presetNameInput = document.querySelector("#preset_name");
+    let durationInput = document.querySelector("#duration");
+    let intervalInput = document.querySelector("#interval");
+    let lootLats = [];
+    let lootLngs = [];
+    let borderLats = [];
+    let borderLngs = [];
+
+    let mapDataValid = (lootLatLngs.length > 0 && markerLatLngs.length > 0 && policeStationLatLng != null);
+    let durationInputValid = (durationInput.value && durationInput.checkValidity());
+    let intervalInputValid = (intervalInput.value && intervalInput.checkValidity());
+    let presetNameValid = (presetNameInput.value && presetNameInput.value.trim() !== "");
+
+    if (!mapDataValid) {
+        showValidationError('Plaats a.u.b. alle pins op de kaart.');
+        return;
+    }
+    if (!durationInputValid) {
+        showValidationError('Vul a.u.b. een valide speelduur in.');
```

Comment:
Hmmm, ik vul een cijfer en druk een paar keer op de knop, dan wordt het validatie-bericht meerdere keren getoond. Voor de gebruiker is een enkel bericht ook al duidelijk genoeg. Dan verander je telkens het validatiebericht ipv nieuwe toevoegen.

![image](https://user-images.githubusercontent.com/60692171/118355651-5d871d80-b571-11eb-85e2-fd48f0d3fb55.png)

Verder weet ik niet of dit bij jouw PR hoort, maar als ik een getal of naam invoer in het input veld, dan krijg ik telkens validatieberichten te zien. Het is voor mij niet duidelijk welke waarde nou wel valide is.

Reply:
Oooh volgensmij worden de validatieberichten getoond van het spelduur die bovenin niet waren ingevuld... 😂 dat is lichterlijk verwarrend

Reply:
Presets horen niet bij deze PR...